common/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -1,25 +1,29 @@
-# pylint: disable=no-member
 """
 Tests for OAuth token exchange views
 """
+
+# pylint: disable=no-member
+
 from datetime import timedelta
 import json
 import mock
 import unittest
 
+import ddt
 from django.conf import settings
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 import httpretty
 import provider.constants
-from provider import scope
 from provider.oauth2.models import AccessToken, Client
 
-from auth_exchange.tests.utils import AccessTokenExchangeTestMixin
 from student.tests.factories import UserFactory
 from third_party_auth.tests.utils import ThirdPartyOAuthTestMixinFacebook, ThirdPartyOAuthTestMixinGoogle
+from .mixins import DOPAdapterMixin, DOTAdapterMixin
```

Comment:
Why are absolute imports required?  Relative implicit imports are allowed by edX python conventions, and recommended in the pink 2-scoops-of-django book.